enterprise/backend/src/metabase_enterprise/library/api.clj

```diff
@@ -0,0 +1,84 @@
+(ns metabase-enterprise.library.api
+  (:require
+   [metabase.api.common :as api]
+   [metabase.api.macros :as api.macros]
+   [metabase.api.routes.common :refer [+auth]]
+   [metabase.collections.core :as collections]
+   [metabase.collections.models.collection :as collection]
+   [toucan2.core :as t2]))
+
+(set! *warn-on-reflection* true)
+
+(api.macros/defendpoint :post "/"
```

Comment:
the REST nerd in me needs to say this should be a PUT since it (can be idempotent), but that isn't important obviously

Reply:
Had it as a POST since we tend to use PUT for updating and this creates it